src/linked-list/doubly-linked-list/DoublyLinkedList.ts

```diff
@@ -17,10 +17,16 @@ export default class DoublyLinkedList {
   private _count: number;
   private _head: DoublyLinkedListNode | null;
   private _tail: DoublyLinkedListNode | null;
+  private _compFunc: (x: any, y: any) => boolean;
+
   constructor() {
     this._head = null;
     this._tail = null;
     this._count = 0;
+    this._compFunc = (x: any, y: any): boolean => {
```

Comment:
Similarly, this doesn't account into whether the user has passed on or no.
Maybe, this? `this._compFunc = compFunc ? compFunc : .......`

Reply:
Similarly, I will make changes to the constructor.